pkgs/top-level/python-packages.nix

```diff
@@ -8833,6 +8833,27 @@ in {
     };
   };
 
+  phpserialize = buildPythonPackage rec {
```

Comment:
Ok. Just to make sure: I need to move this definition into `pkgs/development/python-modules/phpserialize/default.nix` and then use `callPackage` in `pkgs/top-level/python-packages.nix`? 

There are a lot of other Python libraries defined in this file. Are those just some historical remains?